rasa/core/actions/forms.py

```diff
@@ -428,6 +441,17 @@ def _temporary_tracker(
             slots=domain.slots,
         )
 
+    async def _user_rejected_manually(self, validation_events: List[Event]) -> bool:
```

Comment:
there is no need for this to be `async`, no?
```suggestion
    def _user_rejected_manually(self, validation_events: List[Event]) -> bool:
```